src/System.IO.FileSystem/src/System/IO/Win32FileStream.cs

```diff
@@ -72,17 +73,12 @@ internal sealed partial class Win32FileStream : FileStreamBase
         private SafeFileHandle _handle;
         private long _pos;        // Cache current location in the file.
         private long _appendStart;// When appending, prevent overwriting file.
-
-#if USE_OVERLAPPED
-        private static AsyncCallback s_endReadTask;
-        private static AsyncCallback s_endWriteTask;
-        private static Action<object> s_cancelReadHandler;
-        private static Action<object> s_cancelWriteHandler;
-#endif
-
+        
         private StreamAsyncHelper _streamAsyncHelper; // We delegate to this helper for
```

Comment:
So, just to confirm, you want me to remove `USE_OVERLAPPED`? Is `EnableWinRT` no longer relevant? Or does WinRT\UAP now have overlapped support?